internal/pkg/create/create_test.go

```diff
@@ -75,18 +77,71 @@ func TestGetAvailableDirectory(t *testing.T) {
 }
 
 func Test_generateGitZipFileURL(t *testing.T) {
-	url := generateGitZipFileURL("https://github.com/slack-samples/deno-starter-template", "pre-release-0316")
-	assert.Equal(t, "https://github.com/slack-samples/deno-starter-template/archive/refs/heads/pre-release-0316.zip", url, "should return zip download link with branch")
-
-	url = generateGitZipFileURL("https://github.com/slack-samples/deno-starter-template", "")
-	assert.Equal(t, "https://github.com/slack-samples/deno-starter-template/archive/refs/heads/main.zip", url, "should return zip download link with main")
+	tests := map[string]struct {
```

Comment:
note: Reimplemented the tests as a table test and extended the cover. This function now has 100% test coverage. 💯

Reply:
@mwbrooks Huge unlock for test stabilities all around here 🔏 ✨ 

While it'd be so surprising to me, I can imagine missing branches might cause some of the past tests to fail 😉